public/js/components/Scopes.js

```diff
@@ -39,7 +39,7 @@ function getSpecialVariables(pauseInfo, parentName) {
     vars.push({
       name: "<exception>",
       path: parentName + "/<exception>",
-      contents: { value: thrown.toJS() }
+      contents: { value: thrown }
```

Comment:
Hm, that's really interesting: 

here's how i got here:

```patch
diff --git a/public/js/test/integration/todomvc.js b/public/js/test/integration/todomvc.js
index a7e1f40..2e10845 100644
--- a/public/js/test/integration/todomvc.js
+++ b/public/js/test/integration/todomvc.js
@@ -102,7 +102,7 @@ describe("Todo MVC", function() {
    * * pausing on an uncaught expression
    * * reloading while paused and resuming execution
    */
-  it("(Firefox) exception", function() {
+  it.only("(Firefox) exception", function() {
     debugPage("exceptions.html");
     resume();
     scopeAtIndex(0).click();
@@ -111,6 +111,7 @@ describe("Todo MVC", function() {
     resume();
     stepOver();
     stepOver();
+    cy.pause();
     scopeAtIndex(0).click();
     scopeAtIndex(1).contains("unreachable")
```
![screen shot 2016-08-12 at 11 33 14 am](https://cloud.githubusercontent.com/assets/254562/17627790/7c572782-6080-11e6-8f7e-c84f10bbdd45.png)